modin/core/io/text/text_file_dispatcher.py

```diff
@@ -641,6 +649,24 @@ def check_parameters_support(
         if read_kwargs["chunksize"] is not None:
             return False
 
+        skiprows_unsupported = (
+            is_list_like(skiprows_md) and skiprows_md[0] < header_size
+        ) or (
+            callable(skiprows)
```

Comment:
Why are you use `skiprows` and `skiprowd_md` variables at the same time?

Reply:
This is done in that way because in the case of list-like `skiprows` we need already sorted array to get the smallest element (the first element in the `skiprows_md` array) but in the case of callable `skiprows` we can't get `skiprows_md` since it's value already shifted on `header_size` value and we can't get intersection information from it (so we need unshifted `skiprows` callable).